packages/devtools_app/pubspec.yaml

```diff
@@ -9,8 +9,8 @@ version: 2.22.0-dev.4
 repository: https://github.com/flutter/devtools/tree/master/packages/devtools_app
 
 environment:
-  sdk: '>=2.17.0 <4.0.0'
-  flutter: '>=3.0.0'
+  sdk: ">=2.17.0 <4.0.0"
```

Comment:
Single quotes?

Reply:
looks like autoformatting automatically updated this when I saved the file :/